doc/dev/tma.md

```diff
@@ -439,8 +439,32 @@ Please go to [test_tutorial.cpp](../../tests/cpp/test_tutorial.cpp) and search `
 
 ### Examples
 
+#### Example 1: tma-load inputs and vectorize-store output pointwise kernel
+
+Please go to [test_tutorial.cpp](../../tests/cpp/test_tutorial.cpp) and search `TEST_F(Tutorial, VectorizeStorePointwiseTMA)`.
+
+Fusion:
+
+```
+inputs:
+T0[I0, I1] contiguity: T, T
+T1[I0, I1] contiguity: T, T
+
+math:
+T2[I0, I1] = T0[I0, I1] + T1[I0, I1];
+
+outputs:
+T2[I0, I1] contiguity: T, T
+```
+
+schedule:
+
+![example 1: tma-load-inputs, vectorize-store-output pointwise](tma/example1.svg)
```

Comment:
nit:
`Schedule` instead of `schedule`, and `Example 1: TMA-...` instead of `example 1: tma-...`